db/migrate/20171020091213_create_notifications.rb

```diff
@@ -0,0 +1,19 @@
+class CreateNotifications < ActiveRecord::Migration[5.1]
+  def change
+    create_table :notifications do |t|
```

Comment:
We also need `base_path`. Sorry, forgot to put it on the ticket.

Reply:
Ah yep, added that!